android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -129,6 +134,15 @@ protected void onActivityResult(
     }
 
     /**
+     * This method is called if the JS part does not handle the physical back
+     * button press.
+     */
+    @Override
+    public void invokeDefaultOnBackPressed() {
+        super.onBackPressed();
+    }
```

Comment:
* `JitsiMeetActivity` is only one of the two entry points into Jitsi Meet SDK for Android, the other entry point is `JitsiMeetView`. So whatever default behavior upon invoking the back button we provide in `JitsiMeetActivity` we should better provide in `JitsiMeetView` as well for the sake of consistency.

* You and I talked in person today that the above could be achieved by moving the `DefaultHardwareBackBtnHandler` implementation in `JitsiMeetView`. You deduced that it wouldn't be possible because `JitsiMeetView` wouldn't be able to invoke `Activity#onBackPressed()` because it can only invoke its override in extenders. While that's true `Activity#onBackPressed` is documented to only do `Activity#finish()` so, practically, we can achieve that from anywhere. One may argue that that's knowledge of `Activity#onBackPressed()` but then I'd say that (1) that's the official specification of the method and (2) I'm defining the default behavior of `JitsiMeetActivity` and `JitsiMeetView` (if no `JitsiMeetView` customizes it on the JS side) to be to `finish` the associated `Activity`.